style-sheets/primevue-components-overrides.scss

```diff
@@ -1140,4 +1140,133 @@ label {
         display: block;
         margin-top: 1.5rem;
     }
+}
+
+/* ------------------------------------ tabView ----------------------------- */
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-panels {
+    padding: 0;
+    background-color: transparent;
+    z-index: 1;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav {
+    border: none;
+    position: absolute;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header {
+    border-bottom: #ffffff;
+    display: flex;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    display: flex;
+    justify-content: space-between;
+    min-width: 11.16rem;
+    font-weight: 400;
+    background-color: map.get(lightTheme.$light-theme, 'border-subtle-00');
+    border-color: map.get(lightTheme.$light-theme, 'border-subtle-00');
+    padding-bottom: 0.6875rem;
+    border-radius: 0;
+    border-bottom: none;
+    border-right: none;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav-link
+.#{variables.$primevue-prefix}-tabview-title {
+    font-size: 0.875rem;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:first-child
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-top-left-radius: 0.25rem;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:nth-last-child(2)
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-top-right-radius: 0.25rem;
+    border-top-left-radius: 0;
+    border-right: solid 0.0625rem map.get(lightTheme.$light-theme, 'border-subtle-00');
+    border-left: none;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header.#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    font-weight: 700;
+    border-top: solid 0.125rem map.get(
+        lightTheme.$light-theme,
+        'border-interactive'
+    ) !important;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:last-child.#{variables.$primevue-prefix}-highlight
```

Comment:
also here, why it's different than above?

Reply:
This code I was testing with 3 tabs, I think I forgot to remove it